backend/src/services/gameService.ts

```diff
@@ -99,4 +99,27 @@ export const filterGames = async (filter: Filter) => {
 		},
 		where: filter
 	});
-};
\ No newline at end of file
+};
+
+
+export const removeGame = async (gameID : string) => {
+	const game = await prisma.game.findUnique({
+		where: {
+			id: gameID
+		},
+		select: {
+			borrow: true
+		}
+	});
+	if (!game)
+		throw new Error("Game not found");
+	const notReturnedGames = game.borrow.filter(borrow => ((!borrow.returned) && borrow.borrowStart < new Date()));
```

Comment:
I think it makes more sense to have "notReturnedGames" named "borrows" or "borrowedGames"